assets/src/domain/eventEditor/services/apollo/mutations/datetimes/useDatetimeMutationHandler.ts

```diff
@@ -22,7 +22,7 @@ const useDatetimeMutationHandler = (): MutationHandler => {
 	const getMutationVariables = useMutationVariables();
 	const getOptimisticResponse = useOptimisticResponse();
 
-	const mutationHandler: MutationHandler = <MI>(mutationType: MutationType, input: MI) => {
+	const mutationHandler: MutationHandler = (mutationType, input) => {
```

Comment:
won't TS complain about no return type set for function?

Reply:
No, because `mutationHandler` has its type defined